### PR TITLE
Documentation: explain Windows support

### PR DESCRIPTION
**Pull request by penguinolog**
* Fix README.rst validation target in tox.ini

##### Checklist
- [X] I've ensured that similar functionality has not already been implemented
- [X] I've ensured that similar functionality has not earlier been proposed and declined
- [X] I've branched off the `master` or `python-dual-support` branch
- [X] I've merged fresh upstream into my branch recently
- [X] I've ran `tox` successfully in local environment
- [X] I've included docstrings and/or documentation and/or examples for my code (if this is a new feature)

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -12,7 +12,8 @@ commands =
 
 [testenv:readme]
 deps =
-  twine build
```
Comment (penguinolog) [inline tox.ini:15]: inline threated as command instead of list

**Comment by wardi** [comment]
I'm still a little confused about windows support.
- does it depend on WSL?
- does it only work in powershell or does regular cmd work too?
- is raw_display supported or only the curses display module?
- what are the remaining issues with Unicode and supporting other encodings?

Ages ago when I tested on cygwin it seemed to mostly work in that environment/terminal but under `cmd` there was no resizing support. Mingw64 might be another option (cygwin replacement?) for windows support with fewer limitations, I've found it to be a nice way to run bash/ssh when I'm stuck on windows.

**Comment by penguinolog** [comment]
> I'm still a little confused about windows support.
> 
> * does it depend on WSL?
> * does it only work in powershell or does regular cmd work too?
> * is raw_display supported or only the curses display module?
> * what are the remaining issues with Unicode and supporting other encodings?
> 
> Ages ago when I tested on cygwin it seemed to mostly work in that environment/terminal but under `cmd` there was no resizing support. Mingw64 might be another option (cygwin replacement?) for windows support with fewer limitations, I've found it to be a nice way to run bash/ssh when I'm stuck on windows.

1. It not depend on WSL. WSL is linux under windows (literally), it had only issue with RawDisplay due to WSL magic.
2. It works both with `cmd` and `powershell`. Actual `master` branch of `urwid` should execute most UI examples (except with Terminal widget).
3. Supported RawDisplay (we have 2 different versions of RawDisplay: normal and windows with low level Terminal API calls) and curses via windows-curses library (adopted patch from related issue)
4. Windows terminal for a long times unicode and not unicode in the same time - it depends on how data is sent to terminal, as result encoding detection will be always wrong for it.
   If python compiled with unicode support (for windows it's technically always) - it uses Unicode IO and terminal codepage settings does not matter.
   Library windows-curses is distributed compiled for unicode and usage of anything else will cause exceptions (or need to manually compile it from scratch with disabled unicode support).

**Comment by wardi** [comment]
That's great, thanks. So maybe we can also say that under Windows unicode mode is the only one supported.

**Pull request merged**